lib/active_admin_import/import_result.rb

```diff
@@ -32,6 +32,7 @@ def empty?
     def failed_message(options = {})
       limit = options.fetch(:limit, failed.count)
       failed.first(limit).map do |record|
+        record.valid?
```

Comment:
please try to change dependency of activerecord-import to '~> 0.16.1' instead of this line